### PR TITLE
feat: integrate gecko terminal

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

**Review by nguyenhuudungz — APPROVED** [review]

File: src/services/geckoTermial.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      base_token_id: string
+    }
+  }
+  included: [Token, Token]
+}
+
+const geckoTerminalApi = createApi({
+  reducerPath: 'geckoTerminalApi',
+  baseQuery: fetchBaseQuery({
+    // TODO(viet-nv): check prod env
```
Comment (nguyenhoaidanh) [inline src/services/geckoTermial.ts:67]: todo

File: src/components/LiveChart/index.tsx [hunk 2]
```diff
@@ -231,35 +221,32 @@ function LiveChart({
...
-  // const currenciesList = useMemo(
-  //   () => [currenciesState.INPUT, currenciesState.OUTPUT],
-  //   [currenciesState.INPUT, currenciesState.OUTPUT],
-  // )
+  const toggle = useMemo(() => {
+    return (
+      <ProChartToggle
+        activeName={isShowProChart ? 'pro' : 'basic'}
+        toggle={(name: string) => {
+          if (!bothChartError) {
```
Comment (nguyenhoaidanh) [inline src/components/LiveChart/index.tsx:229]: if (!bothChartError && name !== (isShowProChart ? 'pro' : 'basic')) {

File: src/components/TradingViewChart/datafeed.tsx [hunk 3]
```diff
@@ -335,10 +63,11 @@ export const useDatafeed = (currencies: Array<Currency | undefined>, pairAddress
             timezone: Intl.DateTimeFormat().resolvedOptions().timeZone as Timezone,
             exchange: '',
             minmov: 1,
+            // TODO(viet-nv): check pricescale
```
Comment (nguyenhoaidanh) [inline src/components/TradingViewChart/datafeed.tsx:66]: todo

File: src/services/geckoTermial.ts (hunk 1 above)
Comment (viet-nv) [reply src/services/geckoTermial.ts:67]: waiting for geckoterminal open CORS for us
image

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**